sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -32,7 +32,7 @@ def override_all(obj):
 
 # Helpers
 
-def get_livy_kind(language):
+def get_session_kind(language):
```

Comment:
Can we please revert the change in name? Was it done for any particular reason?

Reply:
Yes there is a reason for that change. 

Basically Livy 0.4 onwards has removed "pyspark3" as session kind, which is a breaking change. For sparkmagic, we switch to use the pyspark setting "spark.pyspark.python" to specify python2/python3 instead of the old approach by using the Livy session kind.

As you can see from my change, we are sticking with pyspark/pyspark3 as the session kind definition for sparkmagic, and then translate that to the correct config settings which are sent to Livy when creating a session. I am renaming the method name because this method is now using the new session definition (for sparkmagic), as opposed to livy session definition (which does not have pyspark3 now).

Does it make sense?

Reply:
This change is not merged.

Reply:
0.4 Release Still has `pyspark3` https://github.com/apache/incubator-livy/blob/branch-0.4/core/src/main/scala/org/apache/livy/sessions/Kind.scala

0.5 Removes it https://github.com/apache/incubator-livy/blob/branch-0.5/core/src/main/scala/org/apache/livy/sessions/Kind.scala#L40

Reply:
Has anyone tested this on livy 0.5?